
Mon, 20 Apr 2009 03:32:05 GMT

COMMAND LINE:
/usr/local/src/pypy-dist/pypy-c fannkuch.pypy-4.pypy 10

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "?", line 33, in run_toplevel
  File "fannkuch.pypy-4.pypy", line 11, in <module>
    import multiprocessing as mp
ImportError: No module named multiprocessing
